conf/models.yml

```diff
@@ -21,3 +21,12 @@ model: &least_squares
 
 model: &xgboost
     type: xgboost
+
+model: &averaging
+    type: averaging
+
+model: &decision_tree
+    type: decision_tree
+
+model: &mlp
+    type: mlp
```

Comment:
please end yaml files with a newline -- it makes it easier to handle when combining files programmatically

Reply:
Ok, will do